app/controllers/apartments_controller.rb

```diff
@@ -0,0 +1,49 @@
+class ApartmentsController < ApplicationController
+  before_action :find_apartment, only: [:edit, :update, :destroy, :show]
+
+  def index
+    @apartments = Apartment.all
+  end
+
+  def new
+    @apartment = Apartment.new
+  end
+
+  def create
+    @apartment = Apartment.create(get_params)
+    if @apartment.save
+      redirect_to apartments_path
+    else
+      render 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.